### PR TITLE
Add option for auto upgrade

### PR DESCRIPTION
**Pull request by jennyEckstein**
image

### CODE REVIEW

File: bin/lifion-verify-deps.js [hunk 1]
```diff
@@ -2,11 +2,28 @@
...
+  console.log(
+    [
+      '',
+      'Usage: lifion-verify-deps [options]',
+      '',
+      'Checks existing dependencies for available updates',
+      '',
+      'Options:',
+      '--help           Display this help message and exit',
+      '--auto_upgrade    Automatically run all suggested upgrades'
```
Comment (chendoron) [inline bin/lifion-verify-deps.js]: --auto-upgrade follows a flag naming convention

File: bin/lifion-verify-deps.js [hunk 2]
```diff
@@ -2,11 +2,28 @@
...
+    ].join('\n')
+  );
+}
+
 async function run() {
+  const { auto_upgrade: autoUpgrade } = argv;
```
Comment (chendoron) [inline bin/lifion-verify-deps.js]: `'auto-upgrade':` accordingly

File: lib/index.test.js [hunk 3]
```diff
@@ -178,78 +180,117 @@ describe('lib/index', () => {
...
+
+  test('autoUpgrade modules', async () => {
+    const mock2 = command => {
+      const moduleName = command.match('npm i (.*)')[1];
+      const versions = [olderVersion];
+      if (moduleName === outdatedDep || moduleName === outdatedDevDep) versions.push(newerVersion);
+      return Promise.resolve({ stdout: JSON.stringify(versions) });
+    };
+
+    mockExecAsync.mockImplementationOnce(mock);
```
Comment (chendoron) [inline lib/index.test.js]:
```js
mockExecAsync
  .mockImplementationOnce(mock)
  .mockImplementationOnce(mock)
  .mockImplementationOnce(mock)
  .mockImplementationOnce(mock)
  .mockImplementationOnce(mock2)
  .mockImplementationOnce(mock2);
```

File: .eslintrc [hunk 4]
```diff
@@ -1,5 +1,13 @@
 {
   "extends": "lifion",
+  "overrides": [
+    {
+      "files": ["lib/index.test.js"],
+      "rules": {
+        "jest/no-export": ["warn"]
```
Comment (chendoron) [inline .eslintrc]: Unfortunate usage of export.
We should probably address this with fixtures or something.

@eaviles @jennyEckstein

Comment (eaviles) [reply .eslintrc]: Let's follow it up in a separate PR.

**Pull request merged**